docs/migration_guide.rst

```diff
@@ -1,3 +1,28 @@
+0.8.0 Migration guide
+=====================
+
+Cairo-lang 0.10.1 brings support for `DEPLOY_ACCOUNT` transactions that will completely
+replace currently used `DEPLOY` transactions sometime in the future. You should
+already modify your applications to use new deployment flow.
+
+1. Declare a contract on starknet using `Declare` transaction
+2. Pre-fund the address of new account with enough tokens to cover transaction costs
+3. Send a `DeployAccount` transaction to the pre-funded address
```

Comment:
I don't like the `send a transaction to the address` statement.  Maybe `send a transaction with an account with a pre-funded address`?